tests/AbstractProcessTest.php

```diff
@@ -298,6 +321,56 @@ public function testTerminateWithStopAndContinueSignalsUsingEventLoop()
         $this->assertFalse($process->isTerminated());
     }
 
+    public function testProcessSmallOutput() {
+    	$this->processOutputOfSize(1000);
+    }
+
+    public function testProcessMediumOutput() {
+    	$this->processOutputOfSize(10000);
+    }
+
+    public function testProcessBigOutput() {
+    	$this->processOutputOfSize(100000);
+    }
```

Comment:
These three tests would be better rewritten with the [data provider](https://phpunit.de/manual/current/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers) pattern (using the output size as a single argument).